src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -136,11 +151,11 @@ Graph QuantizeGraph(Graph &&src) {
     NodePtr new_node = Node::Create();
     // If the currently visited node needs quantization, insert a quantize op node before the
     // current node and replace the current node with the quantized version in the new graph.
-    if (NeedQuantize(node, excluded_nodes)) {
-      auto fquantized_op = quantized_op_map[node->op()];
+    auto tmp_node = NeedQuantize(node, excluded_nodes);
+    if (tmp_node) {
+      new_node = tmp_node;
       // If the currently visited node's op registered the FQuantizedOp property, new_node is a
       // quantizated version of a that op, such as quantized_conv2d.
-      new_node = fquantized_op(node->attrs);
```

Comment:
If this line is removed, should we remove the comments as well?

Reply:
Removed.